estela-api/api/views/stats.py

```diff
@@ -224,19 +253,18 @@ def list(self, request, *args, **kwargs):
             spider__in=sid_set, created__range=[start_date, end_date]
         )
 
-        job_collections_names: List[str] = [
+        job_stats_ids: List[str] = [
```

Comment:
Why are we using this notation here? `var_name: type = value`. This is a good practice, but should be generalized.

Reply:
I agree with that. I noticed that in some cases the type is inferred implicitly by django and rest_framework functions, same as typescript so I don't see necessary to put the type explicit (only in some cases), for example this line of code:
```
spider: Spider = Spider.objects.get(sid=kwargs["sid"])
```
Can be replaced by, since the type definition of "get" function is `(method) def get(
    *args: Any,
    **kwargs: Any
) -> Spider` 
```
spider = Spider.objects.get(sid=kwargs["sid"])
```
But for other cases such as defining our own variables outside libraries (django, rest_framework, etc), I see it as a good practice to include it.